src/components/molecules/DefaultAvatars/DefaultAvatars.tsx

```diff
@@ -61,20 +58,16 @@ export const DefaultAvatars: React.FC<DefaultAvatarsProps> = ({
   const avatarImages = useMemo(() => {
     return defaultAvatars.map((avatar, index) => (
       <button
+        data-bem="DefaultAvatars__preview-container"
         key={`${avatar}-${index}`}
-        className={classNames(
-          "DefaultAvatars__preview-container",
-          avatarClassName
-        )}
+        className={classNames("w-12 h-12 mr-1", avatarClassName)}
```

Comment:
Since `avatarClassName` has been there from before this PR, just an

[OOS] We should replace this string prop with one that is more limited and with friendlier ergonomics

Reply:
removed `avatarClassName` since it's not needed anymore: https://github.com/sparkletown/sparkle/pull/2931/commits/fa80b08ff39c2c3413f9336652f0ebbaedeb1398